### PR TITLE
[TT-11699] Update documentation for MDCB 2.5 to master

### PR DESCRIPTION
**Pull request by buger**
Triggered by: sredxny

Included:

Tyk Gateway: false
Tyk Dashboard: false
Tyk MDCB true
Tyk Pump false

Intended for: master
Changes sourced from: release-2.5
Config info generator branch: main

Note: <none> (branch suffix: docs)

JIRA: https://tyktech.atlassian.net/browse/TT-11699

### CODE REVIEW

File: tyk-docs/content/shared/mdcb-config.md [hunk 1]
```diff
@@ -68,216 +68,271 @@ Allows MDCB to use Mutual TLS. This requires to set `server_options.use_ssl` to
...
 Type: `bool`<br />
 
 Allows usage of self-signed certificates when connecting to an encrypted Redis database.
+Deprecated. Use `ssl_insecure_skip_verify` instead.
+
+### storage.timeout
+ENV: <b>TYK_MDCB_STORAGE_TIMEOUT</b><br />
+Type: `int`<br />
+
+Set a custom timeout for Redis network operations. Default value 5 seconds.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Set a custom timeout for Redis network operations. Default value is 5 seconds.
```

File: tyk-docs/content/shared/mdcb-config.md [hunk 2]
```diff
@@ -68,216 +68,271 @@ Allows MDCB to use Mutual TLS. This requires to set `server_options.use_ssl` to
...
+
+### storage.use_ssl
+ENV: <b>TYK_MDCB_STORAGE_USESSL</b><br />
+Type: `bool`<br />
+
+Enable SSL/TLS connection between your Tyk MDCB & Redis.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Enable SSL/TLS connection between Tyk MDCB & Redis.
```

File: tyk-docs/content/shared/mdcb-config.md [hunk 3]
```diff
@@ -68,216 +68,271 @@ Allows MDCB to use Mutual TLS. This requires to set `server_options.use_ssl` to
...
+
+### storage.ssl_insecure_skip_verify
+ENV: <b>TYK_MDCB_STORAGE_SSLINSECURESKIPVERIFY</b><br />
+Type: `bool`<br />
+
+Disable TLS verification
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Disable TLS verification.
```

File: tyk-docs/content/shared/mdcb-config.md [hunk 4]
```diff
@@ -68,216 +68,271 @@ Allows MDCB to use Mutual TLS. This requires to set `server_options.use_ssl` to
...
 
 Sets the batch size for mongo results.
 
 ### analytics.driver
-EV: <b>TYK_MDCB_ANALYTICSCONFIG_DRIVER</b><br />
+ENV: <b>TYK_MDCB_ANALYTICSCONFIG_DRIVER</b><br />
 Type: `string`<br />
 
-Specifies which MongoDB driver should be used by the storage. Valid values are `mongo-go` to use the official mongo driver for Go or `mgo` to use the old driver. The default value is `mgo`.
+Determines the driver used by the storage. It could be `mongo-go` to use the official mongo driver for go or `mgo` to use the old driver.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Use `mongo-go` to use the official mongo driver. Alternatively, use `mgo` to use the old driver.
```

File: tyk-docs/content/shared/mdcb-config.md [hunk 5]
```diff
@@ -380,104 +435,158 @@ Set it to true if you are using a separated analytic storage in the master gatew
...
 Type: `bool`<br />
 
 Allows usage of self-signed certificates when connecting to an encrypted Redis database.
+Deprecated. Use `ssl_insecure_skip_verify` instead.
+
+### analytics_storage.timeout
+ENV: <b>TYK_MDCB_ANALYTICSSTORAGE_TIMEOUT</b><br />
+Type: `int`<br />
+
+Set a custom timeout for Redis network operations. Default value 5 seconds.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Set a custom timeout for Redis network operations. Default value is 5 seconds.
```

File: tyk-docs/content/shared/mdcb-config.md [hunk 6]
```diff
@@ -380,104 +435,158 @@ Set it to true if you are using a separated analytic storage in the master gatew
...
+ENV: <b>TYK_MDCB_ANALYTICSSTORAGE_USESSL</b><br />
+Type: `bool`<br />
+
+Enable SSL/TLS connection between your Tyk MDCB & Redis.
+
+### analytics_storage.ssl_insecure_skip_verify
+ENV: <b>TYK_MDCB_ANALYTICSSTORAGE_SSLINSECURESKIPVERIFY</b><br />
+Type: `bool`<br />
+
+Disable TLS verification
```
Comment (dcs3spp) [inline tyk-docs/content/shared/mdcb-config.md]:
```suggestion
Disable TLS verification.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
@sredxny Thanks for raising the config update for MDCB 2.5.0. I have reviewed with minor suggestions. If you rerun the config generator GHub action with the same parameters, including your TT ticket ID it will update this PR and incorporate the changes.

**Comment by sredxny** [comment]
@dcs3spp check again please. Suggestions has been addressed

**Review by dcs3spp — APPROVED** [review]
Thanks @sredxny I've approved PR. Before merging could you request a peer review? Once peer approved will merge to main branch. Should this be released with 5.3 only?

**Pull request merged**